Functions/Quantification/xASL_quant_FSL.m

```diff
@@ -73,10 +84,14 @@
 	xASL_delete(pathFSLOutput);
     %xASL_adm_DeleteFileList(x.dir.SESSIONDIR, '(?i)^.*basil.*$', 1, [0 Inf]);
     
-    %% 3. Write the PWI as Nifti file for Basil/Fabber to read as input
+    %% 3. Write the PWI4D as Nifti file for Basil/Fabber to read as input
     % FIXME would be good to have a brain mask at this point -> PM: if this would be a brainmask as well, we can skip creating a dummy input image here
     
-    PWI(isnan(PWI)) = 0;
+    PWI4D(isnan(PWI4D)) = 0;
+
+    %% THIS IS TRICKY: WE NEED TO CHECK HERE IF THIS DOESN'T CREATES ZEROS INSIDE THE BRAIN.
```

Comment:
I'm not saying I don't want to fix the NaNs, I'm saying here that the NaNs should perhaps be fixed by neighboring values (especially when they are inside the brain!) rather than replacing them with zeroes.

Reply:
See my fix in 98bb5e453fac67c1d3de0bdcf6b1b5612a95edce. We can add the mask later, but at least we now don't have any erroneous zeroes anymore in the WM (or in the GM vessels!)